### ISSUE TITLE
Add EQs to abnormal tear film morphology and physiology terms

### ISSUE THREAD

**Issue by anna-anagnostop** [mgijax/mammalian-phenotype-ontology#3710]
abnormal tear film morphology MP:0014131

1. Change term label to abnormal tear film composition?

2. Add EQ  'has part' some (composition and ('characteristic of' some 'tear film') and ('has modifier' some abnormal))

3. Currently MP:0014131 has the following parent terms:

- is-a [abnormal anterior eye segment morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0005193)

- is-a [abnormal ocular surface morphology](https://www.informatics.jax.org/vocab/mp_ontology/MP:0013754)

Place under abnormal lacrimal apparatus physiology MP:0013450?

abnormal tear film physiology (currently under abnormal eye physiology MP:0005253 
1. Add EQ 'has part' some 
(functionality and ('characteristic of' some 'tear film') and ('has modifier' some abnormal))

**Comment by anna-anagnostop** [comment]
We should probably obsolete abnormal tear film morphology MP:0014131 and just keep  abnormal tear film physiology  MP:0014132 (as a child of abnormal eye physiology MP:0005253). Will need to look at the annotations to these terms first before we take any action.
